lighthouse-core/report/report-generator.js

```diff
@@ -101,6 +101,15 @@ class ReportGenerator {
     // !value
     Handlebars.registerHelper('not', value => !value);
 
+    // value == value2?
+    Handlebars.registerHelper('if_not_eq', function(conditional, options) {
```

Comment:
which to me is less confusing, but that's partly because I don't grok the handlebars blocks/helpers/context thing at all :)